packages/components/molecules/f-rating/README.md

```diff
@@ -69,14 +69,16 @@ There may be props that allow you to customise its functionality.
+| `starRating`        | `Number`  |   Yes    |     -     | Sets the displayed rating (filled stars). i.e. for `2 out of x`, 2 would be the `starRating`                                                                                                                                                                                                                 | -
+| `maxStarRating`     | `Number`  |    No    |     5     | Sets the maximum number of stars that the rating is set against. i.e. for `x out of 5`, 5 would be the `maxStarRating` You can use any number of stars, so if you currently need 6 all you have to do is pass in 6.                                                                                          | -
+| `starRatingSize`    | `String`  |    No    |  'small'  | Sets the component size. By default the component will use `small`. `Note:` If you find that the size you currently use doesn't fall into one for the four size options here then please move up to the next available size. e.g if you use 22px as your rating size then you should move to 24px (`medium`) | `xsmall = 16px` `small = 20px` `medium = 24px` `large = 32px`
+| `ratingDisplayType` | `String`  |    No    |   null    | Sets the descriptive text which will be displayed next to the component. By default there will be no text.                                                                                                                                                                                                   | `short` `medium` `long`
+| `reviewCount`       | `Number`  |    No    |   null    | Sets the total number of reviews to display. Example: `View 499 reviews` where `499` is the total.                                                                                                                                                                                                           | -
+| `isUserRating`      | `Boolean` |    No    |   false   | Sets the rating as `(You)` to indicate the rating viewed was left by the user.                                                                                                                                                                                                                               | -
+| `shouldAlignRatingLeft`      | `Boolean` |    No    |   false   | When `true` is passed in, the rating display text will be set to the left hand side of the SVG stars.                                                                                                                                                                                                          | -
```

Comment:
I'd be inclined to call this `shouldAlignRatingTextLeft` just to make it really explicit that it's the text part of the rating being aligned

Reply:
Fixed 👍